mbed-client-classic/m2mtimerpimpl.h

```diff
@@ -80,8 +79,23 @@ class M2MTimerPimpl {
      */
     bool is_total_interval_passed();
 
+    /**
+     * @brief Start long period timer
+     */
+    void start_still_left_timer();

```

Comment:
Should this method be private, or is there a place in client side where caller needs to call it explicitly?